chef/cookbooks/crowbar/files/default/knife.rb

```diff
@@ -1,2 +0,0 @@
-node_name "chef-webui"
```

Comment:
-1, this is still used by `cookbook_file "/root/.chef/knife.rb"` on line 178 of the `crowbar::default` recipe, which caused the gate to break.